command/server.go

```diff
@@ -1141,6 +1141,32 @@ func (c *ServerCommand) Run(args []string) int {
 		}
 	}
 
+	isMlockSet := func() bool {
+		// DisableMlock key has been found and thus explicitly set
+		return strutil.StrListContainsCaseInsensitive(config.SharedConfig.FoundKeys, "DisableMlock") ||
+			// mlock is disabled and hence has been explicitly set
+			config.SharedConfig.DisableMlock ||
```

Comment:
What situations could cause the condition above to be skipped, but this condition to be met? Is there another way that the variable can be set to true, other than the config key being specified?